string veh_status
string ble_status
string uwb_status